### PR TITLE
NFDiv-3429 Set app1 & app2 Offline for D8s

### PR DESCRIPTION
**Pull request by BWeeks-JusticeGovUK**
https://tools.hmcts.net/jira/browse/NFDIV-3429

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCase.java [hunk 1]
```diff
@@ -54,7 +54,9 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDet
         data.getLabelContent().setUnionType(data.getDivorceOrDissolution());
 
         var applicant2 = data.getApplicant2();
-        if (JOINT_APPLICATION.equals(data.getApplicationType())) {
+        if (data.isJudicialSeparationCase()) {
```
Comment (amoriartyMOJ) [inline src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCase.java:57]: It seems this if statement does the same as the JOINT_APPLICATION check. Could it be a combined OR statement?

If (isJudicialSeparationCase() || JOINT_APPLICATION.equals(data.getApplicationType()) -> applicant2.setOffline(YES);

Else blah ...

File: src/test/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCaseTest.java [hunk 2]
```diff
@@ -124,4 +125,36 @@ public void shouldNotSetApplicant2OfflineWhenSoleApplicationAndApplicant2HasEmai
...
+        caseData.getApplicant2().setEmail("testapp2@test.com");
+
+        final CaseDetails<CaseData, State> details = new CaseDetails<>();
+        details.setData(caseData);
+        details.setId(1L);
+
+        AboutToStartOrSubmitResponse<CaseData, State> submitResponse = caseworkerCreatePaperCase.aboutToSubmit(details, details);
+
+        assertThat(submitResponse.getData().getApplicant2().getOffline()).isEqualTo(YES);
+    }
```
Comment (amoriartyMOJ) [inline src/test/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCaseTest.java:143]: Possibly assert app1 is also offline? (picky, can be ignored!)

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCase.java (hunk 1 above)
Comment (BWeeks-JusticeGovUK) [reply src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCase.java:57]: I did consider that, and they absolutely could, I just thought it was slightly more obvious/clear this way.

File: src/test/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCaseTest.java (hunk 2 above)
Comment (BWeeks-JusticeGovUK) [reply src/test/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCaseTest.java:144]: A little, but it's a 2 liner.  Done.

**Review by amoriartyMOJ — APPROVED** [review]

**Pull request merged**